.github/workflows/pipeline.yml

```diff
@@ -1,8 +1,9 @@
 name: Pipeline
 
 on:
-  - push
-  - pull_request
+  pull_request:
+  push:
+    branches: main
```

Comment:
is this still relevant?

Reply:
Yes this corrects behavior in PRs so that this doesn't trigger two runs of each job on a push to a PR.

This change will now run on:
- The following changes to PRs:
  - Open
  - Reopen
  - Pushes to the PR
- Pushes to Main

I find the syntax for Github Actions here a bit unclear

Reply:
ok cool